tests/unit/test_advanced.py

```diff
@@ -518,9 +518,9 @@ def test_do_attempt_feedback_partial(self):
 
     def test_do_attempt_keeps_highest_score(self):
         self.assertFalse(self.block.completed)  # precondition check
-        expected_score = 2.0 / 3.0
+        expected_score = 4.0 / 5.0
 
-        self._submit_solution({0: self.ZONE_1, 1: self.ZONE_2})  # partial solution, 0.66 score
+        self._submit_solution({0: self.ZONE_1, 1: self.ZONE_2})  # partial solution, 0.8 score
```

Comment:
There's another comment a couple of lines below that has to be updated: https://github.com/arbrandes/xblock-drag-and-drop-v2/blob/ba9a4e873b00597eb84fa1b42aed8d2580137481/tests/unit/test_advanced.py#L531